S/Skia/build_tarballs.jl

```diff
@@ -0,0 +1,162 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Skia"
+version = v"0.40.0"
+
+# Collection of sources required to complete build
+sources = [
+   GitSource("https://github.com/google/skia.git", "482de011c920d85fdbe21a81c45852655df6a809"),
+   GitSource("https://github.com/stensmo/cskia.git", "3438e6efd3a4f27f43457db675ceb33da30c60cf"),
+   DirectorySource("./bundled"),
+   GitSource("https://chromium.googlesource.com/chromium/src/buildtools.git","1760ff6d7267dd97ae1968c7bee9ce04a2a8489d"),
+   GitSource("https://android.googlesource.com/platform/external/dng_sdk.git","dbe0a676450d9b8c71bf00688bb306409b779e90"),
+   GitSource("https://chromium.googlesource.com/external/github.com/libexpat/libexpat.git","624da0f593bb8d7e146b9f42b06d8e6c80d032a3"),
+   GitSource("https://chromium.googlesource.com/chromium/src/third_party/freetype2.git","5d4e649f740c675426fbe4cdaffc53ee2a4cb954"),
+   GitSource("https://chromium.googlesource.com/external/github.com/harfbuzz/harfbuzz.git","ca3cd48fa3e06fa81d7c8a3f716cca44ed2de26a"),
+   GitSource("https://chromium.googlesource.com/chromium/deps/icu.git","364118a1d9da24bb5b770ac3d762ac144d6da5a4"),
+   GitSource("https://chromium.googlesource.com/codecs/libgav1.git","5cf722e659014ebaf2f573a6dd935116d36eadf1"),
+   GitSource("https://chromium.googlesource.com/chromium/deps/libjpeg_turbo.git","e14cbfaa85529d47f9f55b0f104a579c1061f9ad"),
+   GitSource("https://chromium.googlesource.com/external/gitlab.com/wg1/jpeg-xl.git","a205468bc5d3a353fb15dae2398a101dff52f2d3"),
+   GitSource("https://skia.googlesource.com/third_party/libpng.git","ed217e3e601d8e462f7fd1e04bed43ac42212429"),
+   GitSource("https://chromium.googlesource.com/webm/libwebp.git","845d5476a866141ba35ac133f856fa62f0b7445f"),
+   GitSource("https://chromium.googlesource.com/libyuv/libyuv.git","d248929c059ff7629a85333699717d7a677d8d96"),
+   GitSource("https://android.googlesource.com/platform/external/piex.git","bb217acdca1cc0c16b704669dd6f91a1b509c406"),
+   GitSource("https://chromium.googlesource.com/external/github.com/unicode-org/unicodetools","66a3fa9dbdca3b67053a483d130564eabc5fe095"),
+   GitSource("https://skia.googlesource.com/external/github.com/google/wuffs-mirror-release-c.git","e3f919ccfe3ef542cfc983a82146070258fb57f8"),
+   GitSource("https://chromium.googlesource.com/chromium/src/third_party/zlib","646b7f569718921d7d4b5b8e22572ff6c76f2596"),
+
```

Comment:
Why?  Besides the fact having this many source is a massive code smell and things should be split into multiple packages, we do have already vast majority of those packages as existing JLLs.